wcfsetup/install/files/lib/acp/form/DevtoolsProjectAddForm.class.php

```diff
@@ -49,6 +49,11 @@
  */
 class DevtoolsProjectAddForm extends AbstractFormBuilderForm
 {
+    /**
+     * Defines the minimum WCF version supported by the dev tool.
+     */
+    public const MIN_WCF_VERSION = "3.1.0";
```

Comment:
Please default to private, unless there is a reason for this to be more public.
```suggestion
    private const MIN_WCF_VERSION = "3.1.0";
```